lib/zeta/rspec.rb

```diff
@@ -29,7 +29,7 @@ def self.update_contracts
   def self.run
     Zeta.verbose = false
     ensure_contracts_present
-    Zeta.update_own_contracts
+    Zeta.update_contracts
```

Comment:
Perhaps something like this would also work:

```ruby

  def self.run
    RSpec.describe "Update Zeta infrastructure once" do
      it "download infrastructure" do
        expect{
          Zeta.config
        }.to_not raise_error
      end

      it "download specifications" do
        expect{
          Zeta.update_contracts
        }.to_not raise_error
      end

      describe "validate infrastructure" do
        Zeta.verbose = false
        ensure_contracts_present
        Zeta.update_own_contracts
        Zeta.contracts_fulfilled?(Lacerda::Reporters::RSpec.new)
      end
    end
  end
```